### PR TITLE
Uptime kuma: Transfer rights to yunohost-apps

### PR DESCRIPTION
**Pull request by ketsapiwiq**
Transfer rights to yunohost-apps

### CODE REVIEW

**Comment by frju365** [comment]
Does it need to be on the wishlist as it seem to be already packaged ? :) Wishlist is a place for app which haven't been packaged at all for Yunohost. It doesn't seem to be the case for this app.

**Comment by ketsapiwiq** [comment]
Yes we can totally delete it now, I'll push the revert on my way home

File: pages/02.applications/04.wishlist/apps_wishlist.md [hunk 1]
```diff
@@ -307,7 +307,7 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [umap](https://umap.openstreetmap.fr/) | Cartography software |  | [Package Draft](https://github.com/YunoHost-Apps/umap_ynh) |
 | [unbound](https://nlnetlabs.nl/projects/unbound/about/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/unbound_ynh) |
 | [upmpdcli](https://www.lesbonscomptes.com/upmpdcli/) |  | [Upstream](https://framagit.org/medoc92/upmpdcli) |  |
-| [Uptime Kuma](https://uptime.kuma.pet/) | Self-hosted Uptime Robot | [Upstream](https://github.com/louislam/uptime-kuma) | [Package Draft](https://github.com/ketsapiwiq/uptime-kuma_ynh) |
+| [Uptime Kuma](https://uptime.kuma.pet/) | Self-hosted Uptime Robot | [Upstream](https://github.com/louislam/uptime-kuma) | [Package Draft](https://github.com/YunoHost-Apps/uptime-kuma_ynh) |
```
Comment (yalh76) [inline pages/02.applications/04.wishlist/apps_wishlist.md:310]:
```suggestion
```

**Review by yalh76 — CHANGES_REQUESTED** [review]
Can be removed, as it's now packaged for YunoHost ^^

**Review by frju365 — APPROVED** [review]

**Pull request merged**